chainer/functions/loss/softmax_cross_entropy.py

```diff
@@ -243,9 +243,19 @@ class weight ``class_weight[i]`` is actually multiplied to
             value.
         ignore_label (int): Label value you want to ignore. Its default value
             is ``-1``. See description of the argument `t`.
+        reduce (str): Variable holding a ``str`` which
```

Comment:
`reduce` option is not a `Variable`. So "A string that determines ..." is better.